docker/lnd/start-lnd.sh

```diff
@@ -59,5 +59,6 @@ exec lnd \
     "--$BACKEND.rpchost"="blockchain" \
     "--$BACKEND.rpcuser"="$RPCUSER" \
     "--$BACKEND.rpcpass"="$RPCPASS" \
+    "--rpclisten=0.0.0.0:10009" \
```

Comment:
Add "Do not do this!" command perhaps? 😛  Same for `noseedbackup` above

Reply:
> Add "Do not do this!" command perhaps? stuck_out_tongue Same for `noseedbackup` above

As in add a comment above, saying that this is a very bad idea in prod?